### PR TITLE
test(uns): use http and sdk fixtures

### PR DESCRIPTION
**Pull request by aron-bs**
# Pull Request

## Description

- Add path alias to improve imports
- Update test suites to use `http`/`sdk` fixtures
- Add test cases for uncovered branches
- Add set of local test fixtures to reuse common mocks throughout tests
- Remove unused module mocks from tests (@utils/get-push-id, not needed for these handler tests)

**Related Issue:** https://gdsgovukagents.atlassian.net/browse/FLEX-240

## Type of Change

Please check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor (code change that does not add functionality or fix a bug)
- [x] Code cleanup (formatting, renaming)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so others can reproduce.

- [x] Unit tests
- [x] Integration tests
- [ ] Manual testing (include instructions below)

## Checklist

- [x] I have run the pre-commit
- [x] I have run all necessary tests
- [x] I have updated/added all necessary tests
- [x] I have added or updated documentation
- [x] I have performed a self-review of my code
- [x] I have commented my code in hard-to-understand areas
- [x] I have refactored my code to be more readable, maintainable and removed duplications.
- [x] I have added guards around invalid inputs and edge cases such as nulls and undefined

### CODE REVIEW

File: domains/uns/src/tests/fixtures.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    overrides,
+  );
+export const notification = createNotification();
+
+export const createSecrets = (overrides?: Record<string, string>) =>
+  mergeFixture(
+    { udpNotificationSecret: "test-notification-secret" }, // pragma: allowlist secret
+    overrides,
+  );
+export const secrets = createSecrets();
```
Comment (pazsaragi) [inline domains/uns/src/tests/fixtures.ts:42]: Slight fear stuff like this might clutter the namespace but probably okay

Comment (aron-bs) [reply domains/uns/src/tests/fixtures.ts:42]: I was thinking we could eventually promote these types of common utils to @flex/testing, could then look at coupling them to an event/context namespace instead possibly

**Review by pazsaragi — APPROVED** [review]

**Review by damianpokorski-co — APPROVED** [review]
